### PR TITLE
Like endpoint

### PR DESCRIPTION
**Pull request by dechuck**
New like endpoints, to be able to like and unlike photo for the current user. ( Require write_likes permission )
Retrieve all the likes of a specific user. ( Require public permission )

### CODE REVIEW

File: src/Photo.php [hunk 1]
```diff
@@ -102,4 +102,28 @@ public static function random($filters = [])
...
+	{
+		$like = self::post("photos/{$this->id}/like");
+
+		return true;
+	}
+
+	/**
+	 * Unlike the photo for the current user
+	 * 
+	 * @return [type] [description]
```
Comment (aaronklaassen) [inline src/Photo.php]: Just missing this.

**Comment by aaronklaassen** [comment]
Looks good to me @dechuck

File: src/Photo.php (hunk 1 above)
Comment (dechuck) [inline src/Photo.php]: Doh. Thanks

File: src/User.php [hunk 2]
```diff
@@ -41,6 +42,28 @@ public function photos($page = 1, $per_page = 10)
 		return new ArrayObject($this->photos["{$page}-{$per_page}"]['body'], $this->photos["{$page}-{$per_page}"]['headers']);
 	}
 
+	/**
+	 * Retrieve all the photo liked be a specific user on a given page.
```
Comment (aaronklaassen) [inline src/User.php]: > Retrieve all the photos liked by ...

**Pull request merged**